assets/js/util/tracking/createTrackEvent.js

```diff
@@ -32,6 +32,10 @@ export default function createTrackEvent( config, dataLayerTarget ) {
 			userIDHash,
 		} = config;
 
+		if ( global._gaUserPrefs?.ioo() ) {
```

Comment:
Even though this is rather unlikely, we should also include `ioo` in the chain to be extra safe.
```suggestion
		if ( global._gaUserPrefs?.ioo?.() ) {
```